### DISCUSSION TITLE
Tooltip for touch devices should be activated on click

### DISCUSSION THREAD

**Discussion by benjaminbekes** [primefaces/community#2070] [PrimeNG]
Hello, 

I miss the functionality of a tooltip on mobile (touch) devices. That should be super handy when a user clicks on an element and a tooltip will appear. 

Thanks for a consideration of this enhanced function.

**Comment by albertB4work** [comment]
I support this, i'm trying to add a tooltip on a component that only has inside a split button and I want to show it when it's disabled. Hover and focus events are not working for my case. We need some event like touch or click
